recipes/ipycache/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.1.4" %}
+
+package:
+  name: ipycache
+  version: {{ version }}
+
+source:
+  fn: ipycache-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/i/ipycache/ipycache-{{ version }}.zip
+  md5: c78c6094ffab45b0e33c2f556705fa11
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - ipython
+
+test:  
+  commands:
+    - git clone --depth=50 https://github.com/rossant/ipycache.git source
+    - cd source 
```

Comment:
The reason is there is no testing code in the pypi release and there is no release tags in the repo. So I had to do it that way. I will switch to github zip  in next patch.